pkg/api/server/register_images.go

```diff
@@ -725,8 +725,22 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//    schema:
 	//      type: string
 	//      format: binary
+	//  - in: query
+	//    name: additionalbuildcontexts
+	//    type: array
+	//    items:
+	//      type: string
+	//    default:
+	//    description: |
+	//      Additional build contexts for builds that require more than one context.
+	//      We send both URL and image links through query parameters where we then setup the additional build contexts
+	//      struct. Local additional build contexts are sent as tar files via multipart/form-data.
+	//      (As of version 5.6.0)
```

Comment:
This is the compat endpoint.  Do other clients submit this information this way?